packages/cardano-graphql-services/test/util/stubLogger.ts

```diff
@@ -0,0 +1,12 @@
+import { Logger } from 'ts-log';
+
+// eslint-disable-next-line unicorn/consistent-function-scoping,@typescript-eslint/no-empty-function
+const noOp = () => {};
+
+export const createStubLogger = (): Logger => ({
+  debug: noOp,
```

Comment:
Is there a reason to not just use `jest.fn()`? Wouldn't need to `spyOn` then.

Reply:
https://github.com/input-output-hk/cardano-js-sdk/pull/202/commits/d93a121c626e7c9ce060d575802bc2775cf875e3#diff-190bd16bc1fbf12703a9d5509d897ac1c69b5e8e39dae8a2fe7914820343e22bR3-R9